src/ray/raylet/scheduling/cluster_task_manager_test.cc

```diff
@@ -151,12 +151,14 @@ RayTask CreateTask(
   TaskSpecBuilder spec_builder;
   TaskID id = RandomTaskId();
   JobID job_id = RandomJobId();
+  rpc::JobConfig config;
   rpc::Address address;
   spec_builder.SetCommonTaskSpec(id,
                                  "dummy_task",
                                  Language::PYTHON,
                                  FunctionDescriptorBuilder::BuildPython("", "", "", ""),
                                  job_id,
+                                 config,
```

Comment:
nit:
rpc::JobConfig()

seems easier to read. Otherwise we need check what's config in the code.